docusaurus/docs/quickstarts/zac/installation.md

```diff
@@ -1,26 +1,24 @@
 ---
-title: Installation
+title: Ziti Admin Console
 ---
-# Installing Ziti Administration Console
 
-The Ziti Administration Console (ZAC) is a UI provided by the OpenZiti project which will allow you to configure and 
-explore a [Ziti Network](../../introduction/intro). Installing the ZAC is relatively straightforward 
-and can be accomplished in two basic ways shown below.
+The Ziti Administration Console (ZAC) is a web UI provided by the OpenZiti project which will allow you to configure and 
+explore a [Ziti Network](../../introduction/intro).
 
-## Setting up the Ziti Admin Console
-
-### Prerequisites
+## Prerequisites
 
 It's expected that you're using `bash` for these commands. If you're using Windows we strongly recommend that you install 
 and use Windows Subsystem for Linux (WSL). Other operating systems it's recommended you use `bash` unless you are able to 
-translate to your shell accordingly. 
+translate to your shell accordingly.
+
+You will need `node` and `npm` executables from Node.js v16+. Ubuntu Jammy currently provides v12.

```

Comment:
stating Jammy provides v12 makes me wonder about fedora, or other operating systems. it's better to just say "get the right version for your OS" and leave it at that.